mne_bids/mne_bids.py

```diff
@@ -224,10 +248,22 @@ def _participants_tsv(raw, subject_id, group, fname, verbose):
         Name of group participant belongs to.
     fname : str
         Filename to save the participants.tsv to.
+    write_mode : str, one of ('append', 'overwrite', 'error')
+        How the file should be handled if is exists already.
+        Defaults to 'append'.
+        If write_mode == `append` the previously existing file will have the
+        new data added to it, overwriting any duplicate entries found.
+        If write_mode == `overwrite` the previously existing file will be
```

Comment:
same here

Reply:
ok, I'll fix all those up throughout the code

Reply:
I assume the ones here
https://github.com/mne-tools/mne-bids/blob/a97d6ff0d0cf15b401073658088efc44d4bc4caa/mne_bids/commands/mne_bids_raw_to_bids.py#L57
should be `'`'s, instead of ``` ` ```'s

Reply:
yes agreed